docs/docs/quickstart.md

```diff
@@ -1,40 +1,64 @@
 # Quickstart
 
-[Selecting a DStream Algorithm](algorithm.md)
+For choosing between downstream algorithms, see [Selecting a downstream Algorithm](algorithm.md).
 
 ## Ring Buffer Generalization Intuition
 
-![Traditional ring buffer](buffer-2.png)
-
-Data streams consist of a strictly-ordered sequence of read-once inputs that often exceed available memory capacity. Traditional approaches like circular ring buffers address this limitation by maintaining only the most recent data points and discarding older information. In contrast, Downstream maintains representative, approximate records of stream history by introducing three novel algorithms: 1) "steady" creates evenly spaced snapshots across the entire history, 2) "stretched" preserves important older data points, and 3) "tilted" prioritizes recent information.
 
-![DStream](buffer-1.png)
+Data streams consist of a strictly-ordered sequence of read-once inputs that often exceed available memory capacity. Traditional approaches like circular ring buffers address this limitation by maintaining only the most recent data points and discarding older information.
+![Traditional ring buffer](buffer-2.png)
 
-DStream generalizes this concept with by maintaining representative, approximate records of stream history.
 
+In contrast, downstream maintains representative, approximate records of stream history by introducing three novel algorithms: 1) "steady" creates evenly spaced snapshots across the entire history, 2) "stretched" preserves important older data points, and 3) "tilted" prioritizes recent information.
+![downstream](buffer-1.png)
 
```

Comment:
```suggestion
We provide a more detailed description of available algorithms,and guidance on picking one for your use case in [Selecting a downstream Algorithm](algorithm.md).

```

We should also mention site selection and lookup, and a little more about the ring buffer motiation/analogy. Should probably add this into our docs follow up issue